src/Schema/AbstractIndex.php

```diff
@@ -90,20 +75,14 @@ public function getSort(): array
      */
     public function getColumnsWithSort(): array
     {
-        return array_map(function ($column) {
-            if ($order = $this->sort[$column] ?? null) {
-                return "$column $order";
-            }
-
-            return $column;
-        }, $this->columns);
+        return array_map(
+            fn ($column) => ($order = $this->sort[$column] ?? null) ? "$column $order" : $column,
+            $this->columns
+        );
```

Comment:
```suggestion
        $self = $this;
        return array_map(
            static fn ($column) => ($order = $self->sort[$column] ?? null) ? "$column $order" : $column,
            $this->columns
        );
```
It can be better to use `foreach` construction here.